plugins/inputs/dcos_containers/dcos_containers.go

```diff
@@ -0,0 +1,310 @@
+package dcos_containers
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"math"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
```

Comment:
@branden do you think imports like this should be from the `influxdata` or the `dcos` org? I've used `influxdata` in this PR but maybe that's not right...